ingest/utils/SubmissionError.py

```diff
@@ -0,0 +1,38 @@
+class SubmissionError:
+    """
+    A SubmissionError that can be reported to ingest.
+    """
+
+    def __init__(self):
+        self.type = "http://ingest.data.humancellatlas.org/Error"
+        self.title = ""
+        self.detail = ""
+
+    def getJSON(self):
```

Comment:
Can do this in one line `return self.__dict__`. Ditto everywhere.

Reply:
Awesome, thanks!

Reply:
Awesome, thanks!